### PR TITLE
updated __init__.py to include missing encoders

### LINKED ISSUE #3480: RandomDistributedScalarEncoder and SparsePassThroughEncoder are missing from nupic.encoders __init__.py
All the encoders are in the [__init__.py](https://github.com/numenta/nupic/blob/master/src/nupic/encoders/__init__.py#L22-L35) for nupic.encoders except for
`RandomDistributedScalarEncoder` and `SparsePassThroughEncoder` are missing from nupic.encoders __init__.py

This makes it awkward when you want to import them since you need to do:
```
from nupic.encoders import random_distributed_scalar, ScalarEncoder
rdse = random_distributed_scalar.RandomDistributedScalarEncoder()
scalar_encoder = ScalarEncoder()
```

Ideally you should only need to:

```
from nupic.encoders import RandomDistributedScalarEncoder
rdse = RandomDistributedScalarEncoder()
```

### PR DESCRIPTION
**Pull request by JonnoFTW**
Fixes #3480 

Also fixed absolute package path for the `PassThroughEncoder`

### CODE REVIEW

**Comment by JonnoFTW** [comment]
@rhyolight What did you want changed in multi.py? Do you want all the imports to use the format:

```
from nupic.encoders import ScalarEncoder
```

**Comment by rhyolight** [comment]
Yes, just for consistency's sake

**Comment by rhyolight** [comment]
Interesting test failure (some in all CI platforms)... 🤔 

```
==================================== ERRORS ====================================
_______ ERROR collecting tests/unit/nupic/algorithms/sp_overlap_test.py ________
tests/unit/nupic/algorithms/sp_overlap_test.py:41: in <module>
    from nupic.encoders import scalar
venv/lib/python2.7/site-packages/nupic/encoders/__init__.py:36: in <module>
    from multi import MultiEncoder
venv/lib/python2.7/site-packages/nupic/encoders/multi.py:36: in <module>
    from nupic.encoders import bitsToString
E   ImportError: cannot import name bitsToString
```

**Comment by JonnoFTW** [comment]
@rhyolight I fixed the error, but now I get capnp errors:
```
____________________________________________________________________________________ ERROR collecting tests/unit/nupic/encoders/multi_test.py ____________________________________________________________________________________
tests/unit/nupic/encoders/multi_test.py:37: in <module>
    from nupic.encoders.multi_capnp import MultiEncoderProto
capnp/lib/capnp.pyx:3876: in capnp.lib.capnp._Loader.load_module (capnp/lib/capnp.cpp:78463)
    ???
capnp/lib/capnp.pyx:3855: in capnp.lib.capnp.load (capnp/lib/capnp.cpp:77938)
    ???
capnp/lib/capnp.pyx:3192: in capnp.lib.capnp.SchemaParser.load (capnp/lib/capnp.cpp:68839)
    ???
E   KjException: /home/CSEM/mack0242/.pyenv/versions/2.7.12/lib/python2.7/site-packages/nupic-0.5.8.dev0-py2.7.egg/nupic/encoders/adaptivescalar.capnp:0: failed: Duplicate ID @0xf11485b8c813b489.
E   stack: 0x7f79e78dbf68 0x7f79e82dc3e9 0x7f79e82dbf2a 0x7f79e82d2a1a 0x7f79e82d2f52 0x7f79e82d7346 0x7f79e82d743b 0x7f79e82d74fc 0x7f79e82d7544 0x7f79e82b7ff9 0x7f79e82b7246 0x7f79e82bb4ac 0x7f79e82cf945 0x7f79e82d72ce 0x7f79e82d056e 0x7f79e82d0877
```
Are these errors acceptable?

**Comment by rhyolight** [comment]
@JonnoFTW Can you merge in master and see if the builds pass?

**Comment by JonnoFTW** [comment]
@rhyolight  can you have a look at this now?

File: src/nupic/encoders/multi.py [hunk 1]
```diff
@@ -20,21 +20,21 @@
...
+from nupic.encoders import DateEncoder
+from nupic.encoders import LogEncoder
+from nupic.encoders import CategoryEncoder
+from nupic.encoders import SDRCategoryEncoder
+from nupic.encoders import DeltaEncoder
+from nupic.encoders import ScalarSpaceEncoder
+from nupic.encoders import PassThroughEncoder
+from nupic.encoders import SparsePassThroughEncoder
+from nupic.encoders import CoordinateEncoder
+from nupic.encoders import GeospatialCoordinateEncoder
```
Comment (rhyolight) [inline src/nupic/encoders/multi.py]: According to our [style guide](http://nupic.docs.numenta.org/prerelease/contributing/python-style-guide.html#imports), please import all encoders enclosed in parentheses to allow multiple lines.

**Comment by rhyolight** [comment]
There is one big problem with this. With all these encoder name changes, we'll need to update the API doc source. I'll put this in a PR against this one.

**Comment by rhyolight** [comment]
Thanks Jonathan!

**Pull request merged**